libusb/core.c

```diff
@@ -682,6 +682,10 @@ ssize_t API_EXPORTED libusb_get_device_list(libusb_context *ctx,
 		usbi_mutex_unlock(&ctx->usb_devs_lock);
 	} else {
 		/* backend does not provide hotplug support */
+		if (!usbi_backend->get_device_list) {
+	          len = -1;
+		  goto out;
+		}
```

Comment:
I didn't test this on all of Mac/Linux, but pretty sure it's a safe change as we aren't using `lib{usb,uvc}_get_device_list()`  currently.